1.Core/Helpers/Logger.cs

```diff
@@ -0,0 +1,37 @@
+﻿using NUnit.Framework;
+using Serilog;
+
+namespace Todoly.Core.Helpers;
+public static class Logger
+{
+    private static ILogger[]? _logger;
+
+    public static ILogger[] Instance
+    {
+        get
+        {
+            if (_logger == null)
+            {
+                _logger = CreateLoggersForTest(TestContext.CurrentContext.Test.Name);
+            }
+
+            return _logger;
+        }
+        set => _logger = value;
+    }
+    public static ILogger[] CreateLoggersForTest(string? testName)
```

Comment:
By returning an array of loggers you don't have another option than to use a for loop every time you want to log something, which results in a lot of repeated code. I suggest creating a Debug, Error, and Information static methods that will log something on both loggers, something like this:
```csharp
public static void Information(string message)
{
    Instance._testLogger.Information(message);
    Instance._generalLogger.Information(message);
}
```

Notice that you will need to store the `_testLogger` and the `_generalLogger` loggers as private fields in the singleton. Calling this method on the hooks and UIElement catch block would look like this: `Logger.Information("...")`. In case the singleton instance is not accesible on a static method, the method should become an instance method and calling it would be `Logger.Instance.Information("...")`